### ISSUE TITLE
in fancy editor, editing text using cut copy paste sometimes results in text loss and mangling

### ISSUE THREAD

**Issue by xenek** [honestbleeps/Reddit-Enhancement-Suite#5427]
- Browser: **[Firefox 107 on Windows 10]**
- RES version:  **Reddit Enhancement Suite v5.22.10**
- reddit beta?: **no**

*What's the problem?*

Over I think, at least the last six months, but probably longer, when I use cut/paste I often loose text, and the display is corrupted and small blocks of text are lost. It only occurs when editing in the fancy editor. When I use markdown mode, I have no problems. It usually only happens on long comment replies, and is intermittent, but today it is happening on a tiny single-line response as well. Or maybe I don't edit those so much I've noticed it there before. 

*Are there any related posts in https://github.com/honestbleeps/Reddit-Enhancement-Suite/issues or /r/RESissues?*

I didn't see it reported elsewhere here, I searched for keyword 'fancy' - so I guess it's specific to a combination of settings or hardware or software I have, or is rarely reported. I figured I'd log it so that if someone else has the issue, they have a guide or resolution. 

But it does look like others have this issue, and it might be related to firefox.
https://www.reddit.com/r/bugs/comments/oydg2n/when_commenting_fancy_pants_editor_bugs_out_when/
https://www.reddit.com/r/firefox/comments/te2fie/is_new_reddits_fancy_pants_editor_reply_editor/
https://www.reddit.com/r/ModSupport/comments/ou7f8v/copypaste_in_the_fancy_pants_editor_has_been/

*What other browser extensions are installed?*

Ublock Origin, Ecosia, Lastpass, Pocket.

*Resolution applied?*

I have no resolution, I usually take a breath in and swear. Sometimes I blame random people cursing under my breath. I didn't blame RES though, but I guess this bug report is a long complicated less confronting way of saying 'oh no'. (insert your favorite four letter word instead of 'oh no')
<img width="313" alt="1 type short line before doing copy paste" src="https://user-images.githubusercontent.com/27728244/202843822-a553a54a-33e7-4d70-a398-187c6c093fe4.PNG">
<img width="311" alt="2 cut some of the text" src="https://user-images.githubusercontent.com/27728244/202843826-fb9bffe6-2fd8-4258-84b2-5e1a5748e23f.PNG">
<img width="314" alt="3 paste at the end and the line mangles" src="https://user-images.githubusercontent.com/27728244/202843827-65174d05-302b-4f83-898d-ec2f1438e8ed.PNG">
<img width="314" alt="4 click markdown button and its all gone" src="https://user-images.githubusercontent.com/27728244/202843828-c0afce49-8044-4b04-a78f-31c54f3d0edf.PNG">

**Comment by creesch** [comment]
This doesn't seem RES related though? The fancy editor is something reddit provides.

**Comment by xenek** [comment]
True, but I thought it's best to log it here as it may complicate RES functionality, or the RES team might be able to help the Reddit team (if they want to fix it). Also, some people might blame it on RES without realizing it's a Reddit core fault, to do with what appears from the other posts, to be due to using Firefox. Maybe it's causing RES to loose users, if RES users tend to use Firefox and the firefox plugin.
